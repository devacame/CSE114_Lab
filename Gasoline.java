import java.util.Scanner;

public class Gasoline {
    public static void main(String[] args) {
        int minutes = 0;
        double mph = 0.0, mpg = 0.0;
        Scanner input = new Scanner(System.in);

        while (minutes < 0) {
            System.out.print("Enter the number of minutes: ");
            minutes = input.nextInt();
        }
        while (mph < 0) {
            System.out.print("Enter the constant speed of the car(m/h): ");
            mph = input.nextDouble();
        }
        while (mpg < 0) {
            System.out.print("Enter the fuel efficiency of the car (m/g): ");
            mpg = input.nextDouble();
        }
        input.close();

        System.out.println("Consumed gas for the period (in gallons): " + minutes / 60.0 * mph / mpg);
    }
}
