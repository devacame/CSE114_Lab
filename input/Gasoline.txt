75
35.2
20.6