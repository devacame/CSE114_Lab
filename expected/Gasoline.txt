Consumed gas for the period (in gallons): 2.1359223300970873
